$OpenBSD: patch-stickynotes_stickynotes_ui,v 1.2 2010/04/23 13:58:34 ajacoutot Exp $

Fix crash of stickynotes. Patch from Open Solaris'
gnome-applets-07-stickynotes-crash.diff

--- stickynotes/stickynotes.ui.orig	Sun Mar 28 07:29:51 2010
+++ stickynotes/stickynotes.ui	Fri Apr 23 11:28:40 2010
@@ -593,7 +593,7 @@
               </packing>
             </child>
             <child>
-              <object class="GtkVBox" id="vbox1">
+              <object class="GtkVBox" id="vbox3">
                 <property name="visible">True</property>
                 <property name="orientation">vertical</property>
                 <child>
@@ -613,7 +613,7 @@
                   </packing>
                 </child>
                 <child>
-                  <object class="GtkLabel" id="label1">
+                  <object class="GtkLabel" id="label2">
                     <property name="visible">True</property>
                     <property name="xalign">0</property>
                     <property name="label" translatable="yes">This cannot be undone.</property>
